knowledge/responses/what_about_protein.md

```diff
@@ -1,4 +1,4 @@
-One of the most prevalent myths of veganism is that vegan diets are lacking in enough protein to be healthy, and the only way to get healthy amounts of protein naturally is through the consumption of meat, dairy, and/or eggs. This common myth has existed for a long time, but is based on flawed logic, misguided understanding of nutrition, and a disregard of plentiful evidence to the contrary.
+The myth that it is impossible or difficult to consume enough protein in vegan diets is incorrect. It is a long standing misconception that stems from a lack of understanding of science and nutrition.
 
 To start, there are several athletes who have sustained or increased their performances under vegan diets. Several athletic and strength world record holders completed their records as vegans, and more athletes, powerlifters, and bodybuilders are seeing continued improvements after switching to a vegan diet [1], proving that a vegan diet provides sufficient protein to maintain an athletic lifestyle. Although some may point out that these athletes only get enough protein through supplementaiton, it is important to note that supplements are nothing new for athletes. In reality, vegan athletes supplement because they are athletes, not because they are vegan. All athletes supplement [2]. For the average person, a proper vegan diet would not necessitate protein supplementation.
```

Comment:
i think we can't say "vegan diet provides sufficient protein to maintain an athletic lifestyle" and then say that athletes have to supplement. 

The thing about athletes and protein is that if they're trying to get bigger, they have to eat more protein, and they supplement in whatever diet because the calorie to protein ratio in food is just not ideal for their training. If they're not trying to get bigger they supplement probably to increase recovery - or sometimes because of misconceptions about protein. If they're not training they don't have to supplement because they are not going to get bigger or going to have muscle soreness or other injuries.

Maybe the thing to say here is that look - you can eat vegan and still perform at peak athletic levels, even their supplemented protein came from plants, not animal products. And they still get enough to function just like their non-vegan counterparts. It is not more difficult for them to get nutrients.

Reply:
Addressed.